coderedcms/views.py

```diff
@@ -61,7 +63,13 @@ def search(request):
                 model = ContentType.objects.get(
                     model=search_model
                 ).model_class()
-                results = results.type(model)
+                # Workaround for Wagtail MySQL search bug.
+                # See: https://github.com/wagtail/wagtail/issues/11273
+                backend = get_search_backend()
+                if type(backend) == MySQLSearchBackend:
+                    results = model.objects.live()
+                else:
```

Comment:
Is there a disadvantage to always doing `model.objects.live()`?

Reply:
`.type(Model)` will query on Model, and also anything that inherits from Model. It is also the recommended approach in the docs. Doing `Model.objects.live()` will of course ONLY query on Model, not descendants.